stan/math/rev/scal/fun/fma.hpp

```diff
@@ -121,7 +121,8 @@ inline var fma(const var& a, const var& b, const var& c) {
  * @param c Summand.
  * @return Product of the multiplicands plus the summand, ($a * $b) + $c.
  */
-inline var fma(const var& a, const var& b, double c) {
+template <typename Tc, typename = enable_if_arithmetic<Tc>>
+inline var fma(const var& a, const var& b, Tc c) {
```

Comment:
Think below perfect forwarding is fine? Though idt it matters for an integral type

```suggestion
inline var fma(const var& a, const var& b, Tc&& c) {
```